dataservices/helpers.py

```diff
@@ -102,6 +119,19 @@ def __init__(self):
         self.un_urban_pop = pandas.read_csv('dataservices/resources/urban_population_annual.csv')
         self.year = datetime.today().year
 
+        # Lets make the counties columns consistent with out own source
+        self.un_male_pop_data = self.map_country_data(self.un_male_pop_data)
+        self.un_female_pop_data = self.map_country_data(self.un_female_pop_data)
+        self.un_rural_pop = self.map_country_data(self.un_rural_pop)
+        self.un_urban_pop = self.map_country_data(self.un_urban_pop)
+
+    def map_country_data(self, country_data):
+        # This function is used to map the DS to match our naming convention for countries
+        country_data.country_name = country_data.country_name.apply(
+            lambda x: self.country_map.get(x) if self.country_map.get(x) is not None else x
```

Comment:
Can we get away with just this, using `x` as the default val on a key miss?

```suggestion
            lambda x: self.country_map.get(x, x)
```